Source/WebCore/testing/Internals.cpp

```diff
@@ -1659,7 +1663,7 @@ void Internals::selectColorInColorChooser(HTMLInputElement& element, const Strin
 
 ExceptionOr<Vector<AtomString>> Internals::formControlStateOfPreviousHistoryItem()
 {
-    HistoryItem* mainItem = frame()->loader().history().previousItem();
+    auto* mainItem = frame() ? frame()->loader().history().previousItem() : nullptr;
```

Comment:
I’d think this would need to be `RefPtr` instead of `auto*` in our new safer C++ model.

Reply:
Done